packages/cpp/src/browser/cpp-preferences.ts

```diff
@@ -90,7 +85,6 @@ export const cppPreferencesSchema: PreferenceSchema = {
 export class CppConfiguration {
     'cpp.buildConfigurations': CppBuildConfiguration[];
     'cpp.experimentalCompilationDatabaseMap': boolean;
```

Comment:
Since you remove the experimental commands, what about this one: Do you still need it ?
cpp.experimentalCompilationDatabaseMap

Reply:
Yes it is still needed. The preference enables/disable an experimental way to handle multiple root workspaces using clangd.